src/components/MultiLink/Legend.vue

```diff
@@ -462,13 +502,17 @@ export default Vue.extend({
         <br>
       </div>
 
+      <div v-if="cleanedNodeVariables.size === 0">
+        No Node Attributes To Visualize
+      </div>
```

Comment:
I think you should combine this with the v-for block above as its v-else. That is, display this block if there are no cleaned node variables; otherwise, display the v-for block. Might make this section easier to understand for the next person who comes along.

Reply:
The titlecased message seems weird; can it just be capitalized normally?

Reply:
(Same comments below, for the cleaned link variables.)

Reply:
Fixed in 8f95444